src/components/OSCALBackMatter.js

```diff
@@ -19,8 +19,6 @@ export const OSCALBackMatterCard = styled(Card)(
 `
 );
 
-// TODO: Temporary fix for missing media type (https://github.com/GSA/fedramp-automation/issues/103)
-// Uses file extension instead
```

Comment:
While the issue upstream has potentially been fixed, we have not removed our workaround. We still need to do so. We probably want to track this in a separate ticket, unless you want to talk about how to remove the specific workaround here.

Reply:
I added the `TODO` back into the code base and here is the ticket I made for it: https://github.com/EasyDynamics/oscal-react-library/issues/512